createContacts.html

```diff
@@ -35,7 +35,7 @@
 					</div>
 				</div>
 				<button type="button" id="addContactButton" class="button" onclick="addContact();">Add Contact</button>
-				<span id="loginResult"></span>
+				<span id="contactAddResult"></span>
 			</div>
```

Comment:
These error/confirmation messages still need some formatting in the CSS @CathyChian